detox.config.js

```diff
@@ -0,0 +1,77 @@
+/** @type {Detox.DetoxConfig} */
+module.exports = {
+  logger: {
+    level: process.env.CI ? 'debug' : undefined,
+  },
```

Comment:
What's the behavior for a non-CI environment? Does it default to `info`? 
I can see Detox docs have this line written slightly different - https://wix.github.io/Detox/docs/config/overview

Reply:
This appears to be the same as the Detox example app: https://github.com/wix/Detox/blob/master/examples/demo-react-native/detox.config.js